src/test/java/de/neuland/jade4j/benchmark/TemplateBenchmark.java

```diff
@@ -0,0 +1,66 @@
+package de.neuland.jade4j.benchmark;
+
+import de.neuland.jade4j.JadeConfigurationCaffeine;
+import de.neuland.jade4j.template.ClasspathTemplateLoader;
+import de.neuland.jade4j.template.JadeTemplate;
+import org.openjdk.jmh.Main;
+import org.openjdk.jmh.annotations.*;
+
+import java.io.StringWriter;
+import java.io.Writer;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+
+@Fork(1)
+@Warmup(iterations = 10)
+@BenchmarkMode(Mode.Throughput)
+@Measurement(iterations = 5)
+@State(Scope.Benchmark)
+public class TemplateBenchmark {
+
+    ClasspathTemplateLoader templateLoader = new ClasspathTemplateLoader();
+
+    List<String> books = Arrays.asList("booka", "bookb", "bookc");
+
+    @Param({ "0", "1" })
+    public int templateId;
+
+    HashMap<String, Object> model = new HashMap<>();
+
+    JadeConfigurationCaffeine jadeCaffeine = new JadeConfigurationCaffeine();
+
+    JadeConfigurationCaffeine jade = new JadeConfigurationCaffeine();
```

Comment:
You are testing JadeConfigurationCaffeine against JadeConfigurationCaffeine. I think this should be Jade Configuration().